### ISSUE TITLE
Incorrect warning - Sentence must end with a period

### ISSUE THREAD

**Issue by khrome83** [gajus/eslint-plugin-jsdoc#10]
The rule `jsdoc/require-description-complete-sentence` requires a `.` at the end of a sentence. But it has issues with the following comment block. 

Rules like `@function` and `@memberof` should not trigger the description warning.

``` javascript
/**
 * Creates a document node.
 * 
 * @function createDocument
 * @memberof TreeAdapter
 * 
 * @returns {ASTNode<Document>} Document node.
 */
```

It is worth pointing out that if `.` are added to the lines without them, the warning is still generated.

**Comment by gajus** [comment]
This is a more serious bug. It is complaining about:

```js
/**
 * @returns {ASTNode<Document>} Document node.
 */
```

Specifically, https://github.com/yavorskiy/comment-parser is parsing this as:

```js
{
    tag: 'returns',
    type: 'ASTNode<Document>',
    name: 'Document',
    optional: false,
    description: 'node.',
    line: 3,
    source: '@returns {ASTNode<Document>} Document node.'
}
```

It wrongly assumes that "Document" is the name property of the tag.

I will see what can be done about it.

**Comment by gajus** [comment]
It is a known behaviour, https://github.com/yavorskiy/comment-parser/issues/21.

> As for natively supported return, idea was to build generic parser agnostic to actual tags

ie., parser assumes that all tags have syntax `[tag] [type] [name] [description]`.

**Comment by gajus** [comment]
Version 2.2.1 released.
